source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -0,0 +1,86 @@
+
+### Bug Fixes
+ - Cleaned up the config store on server initialization errors.
+ - Fixed an issue where permissions did not grant read and/or write access to the Global Relay configuration settings.
+ - Fixed an issue where the site configuration ‘’Read only’’ permission did not make the "Notice" section as read-only for the System Manager.
+ - Fixed an issue where importing Client4 in a node server caused an exception due to rudder modules.
+ - Fixed an issue where LDAP ‘’FirstLoginSync’’ didn't close LDAP Session.
+
+### Known Issues
+ - System Manager does not have access to Billing section and sees blank screen.
+ - Cloud > "Tips & Next Steps" should not show "Explore channels" section for guest users.
+ - System Roles shows License and Environment as possible permissions but are always hidden in Cloud
+
+## Release 2020-12-03
+
+### Bug Fixes
+  - Disabled the xmlsec1-based SAML library in favor of the re-enabled and improved SAML library.
+
+## Release 2020-11-24
+
+### Highlights
+
+#### Modify new Admin roles permissions in the System Console (Beta)
+ - Added a user interface for the existing new system roles functionality including a new permission to read and write system_roles.
+
+#### Pre-package and enable incident management v1.1.1
+
+### Improvements
+
+#### User Interface
+ - @-autocomplete results are now prioritized based on recency and thread activity.
+ - File attachments below size of 10 now show fractions.
+ - The formatting of the channel header change message was improved.
+ - Team invite workflow now shows BOT tags when the search returns Bot users.
+ - Added the ability to zoom in/out of PDF files.
+ - Added support for 16x16 base64 encoded mini images to use with progressive rendering.
+
+#### Notifications
+ - Channel-wide mentions are now automatically disabled when a user mutes a channel.
+
+#### Integrations
+ - Updated icon_emoji field in incoming webhooks to allow for emojis to also be specified with surrounding colons.
+ - Dynamic auto-completion is now supported for built-in slash commands.
+ - Added plugin hooks for ReationHasBeenAdded and ReactionHasBeenRemoved.
+
+#### Administration
+ - Added the ability to load a set of configuration custom defaults from a MM_CUSTOM_DEFAULTS_PATH environment variable.
+ - Added AWS metering service support.
+ - Added the ability to retrieve compliance files from the System Console.
+
+### Bug Fixes
+ - Fixed an issue with broken link previews for twitter links.
+ - Fixed an issue where editing a post did not submit with CMD+ENTER.
+ - Fixed an issue where users were able to create or edit slash commands to have more than two slashes in the URL.
+ - Fixed an issue where resized emojis got overwritten with original data.
+ - Fixed an issue where the sidebar category "More" menu was not shown when hovering over a long category name.
+ - Fixed an issue where a received direct message did not show up on the sidebar if it wasn't previously created.
```

Comment:
This bullet point is unclear.
- What wasn't previously created -- the sidebar or the received DM?

Reply:
The DM channel - adjusted the wording.